smaily-for-wp.php

```diff
@@ -142,7 +144,7 @@ function smaily_subscribe_callback() {
 	$form_values = [];
 	// Add custom form values to Api request if available.
 	foreach ( $params as $key => $value ) {
-		if ( array_key_exists( $key, $array ) ) {
+		if ( array_key_exists( $key, $array ) || $key === 'action' ) {
 			continue;
 		} else {
 			$form_values[ $key ] = $value;
```

Comment:
Apply `sanitize_text_field` to `$value` like it is being done in `smaily_nojs_subscribe_callback`. Otherwise data sent to Smaily can be different depending on how form was submitted.